### ISSUE TITLE
.toml format for c8y-log-plugin and c8y-configuration-plugin is different

### ISSUE THREAD

**Issue by mbay-ODW** [thin-edge/thin-edge.io#1209]
In the current implementation 0.7.1 the .toml files for configuration and log plugin is different.

c8y-log-plugin:

![Bildschirmfoto 2022-06-21 um 09 06 07](https://user-images.githubusercontent.com/58465677/174737443-ac3432e1-9784-42ad-9c16-96baec5cba52.png)

c8y-configuration-plugin:

![Bildschirmfoto 2022-06-21 um 09 06 32](https://user-images.githubusercontent.com/58465677/174737491-1e14dd99-56e9-4c5c-9e05-e6fa38e97982.png)

My recommendation would be to use the format of the configuration-plugin since its already capable of allowing multiple log files. This will be the case for productive usage anyway and I don't see why there should be two different file formats.

**Comment by didier-wenzek** [comment]
Both are valid alternatives syntax for list of records. This is just a matter of preference.

The log plugin configuration can be written as:

```
files = [
  type = 'software-management'
  path = '/var/log/tedge/agent/software-*'
]
```

That said, I agree that this is confusing and that we should use the same syntax for all examples and documentation.

**Comment by rina23q** [comment]
Read [Array of Tables](https://toml.io/en/v1.0.0#array-of-tables). Also, the syntax of c8y-log-plugin allows multiple entries. For example, you can write,

```toml
[[files]]
path = "/var/log/tedge/agent/software-*"
type = "software-management"

[[files]]
path = "/var/log/tedge/agent/another-software-*"
type = "another-software-management"
```

It is equal as

```toml
files = [
  { path = "/var/log/tedge/agent/software-*", type = "software-management"  },
  { path = "/var/log/tedge/agent/another-software-*", type = "another-software-management" },
]
```

**Comment by rina23q** [comment]
Use the JSON like format for c8y-log-plugin. Change the code of c8y-log-plugin. (doc is fine https://thin-edge.github.io/thin-edge.io/html/howto-guides/023_c8y_log_plugin.html)

**Comment by PradeepKiruvale** [comment]
#1223

**Comment by rina23q** [comment]
We fixed it by #1223. Now the content of files created by `--init` options are following.

**c8y-configuration-plugin.toml**

```toml
# Add the configurations to be managed by c8y-configuration-plugin

files = [
#    { path = '/etc/tedge/tedge.toml' },
#    { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf', type = 'c8y-bridge.conf' },
#    { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf', type = 'tedge-mosquitto.conf' },
#    { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto.conf' },
#    { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o444 }
]
```

**c8y-log-plugin.toml**

```toml
files = [
    { type = "software-management", path = "/var/log/tedge/agent/software-*" },
]
```

@mbay-ODW 
If you are happy with the change, can you close the issue? You can also wait until the fix is shipped as a part of 0.7.3 (will be in two weeks) if you want to confirm the fix in order to decide to close the issue.
